README.md

```diff
@@ -15,8 +15,7 @@ This module returns the ID for the name of a resource group. It can also create
 Existing resource group
 ```hcl
 module "resource_group" {
-  # Replace "main" with a GIT release version to lock into a specific release
-  source                       = "git::https://github.com/terraform-ibm-resource-group/terraform-ibm-resource-group.git?ref=main"
+  source  = "../.."
```

Comment:
This is the wrong format for a README. It should be source and version = latest with the comment updated.